esx_service/cli/vmdkops_admin.py

```diff
@@ -415,6 +415,10 @@ def commands():
                     'help': 'Volume to set options for, specified as "volume@datastore".',
                     'required': True
                 },
+                '--tenant': {
+                    'help': 'Name of the tenant which volume to set belongs to.',
```

Comment:
Nit: Suggest to rephrase a little bit: "Name of the tenant the volume belongs to"